plugin/src/makeWorklet.ts

```diff
@@ -301,10 +79,10 @@ export function makeWorklet(
   // comment after the function that gets included here, and then the closing
   // bracket would become part of the comment thus resulting in an error, since
   // there is a missing closing bracket.
-  const code =
+  codeObject.code =
     '(' + (isObjectMethod(fun) ? 'function ' : '') + codeObject.code + '\n)';
 
-  const transformed = transformSync(code, {
+  const transformed = transformSync(codeObject.code, {
```

Comment:
This change affects the behavior of the plugin, what's the motivation and what are the effects?

Reply:
We don't use `codeObject.code` anywhere else so I thought's its just logical to replace it instead of making another variable, especially since later we use for example `codeObject.map`.